.github/ISSUE_TEMPLATE/missing-layer-type.md

```diff
@@ -7,8 +7,10 @@ assignees: ''
 
 ---
 
+# Missing Layer Type
+Please fill out these questions:
 - Name of layer type:
 - Is this a PyTorch or a TensorFlow layer type:
-- Your version of coremltools:
-- Your version of PyTorch/TensorFlow:
+- The version of coremltools that you are using:
```

Comment:
I like the current wording better. It's more concise.

Reply:
Ok